libc/newhdrgen/yaml_combined/sys_wait.yaml

```diff
@@ -0,0 +1,33 @@
+header: sys-wait.h #sys/wait.h
```

Comment:
delete comment

Reply:
will delete once all the yaml files are finalized in case any changes occur in between.